conda/gateways/README.md

```diff
@@ -0,0 +1,25 @@
+## `conda.gateways`
+
+Gateways isolate interaction of conda code with the outside world.  Disk manipulation,
+database interaction, and remote requests should all be through various gateways.  Functions
+and methods in `conda.gateways` must use `conda.models` for arguments and return values.
+
+Conda modules importable from `conda.gateways` are
```

Comment:
All of these lines as well, please, need colons.

Reply:
```suggestion
Conda modules importable from `conda.gateways` are:
```